travis.sh

```diff
@@ -2,5 +2,13 @@
 
 set -uexo pipefail
 
-make -f posix.mak all DMD="$(which $DMD)"
-make -f posix.mak test DMD="$(which $DMD)"
+~/dlang/install.sh install ldc
+
+~/dlang/install.sh list
+
+LDMD2=$(find ~/dlang -type f -name "ldmd2")
+
+make -f posix.mak all DMD=$(which dmd)
+make -f posix.mak test DMD=$(which dmd) \
+    RDMD_TEST_COMPILERS=dmd,$LDMD2 \
```

Comment:
Always use quotes for variables ( I recommend the shellcheck linter)

Reply:
You were quick, I'd have fixed that :-P

Reply:
Travis always uses the same paths as do we on the installer side, so it wasn't really blocking this PR ;-)
(And I generally prefer to avoid PRs getting stalled because of such tiny nits).
Thanks for the quick follow-up.